web/Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM node:12-alpine
 WORKDIR /usr/src/app
 RUN apk update && apk add --virtual bash coreutils
+RUN apk add --no-cache git
+RUN git --version
```

Comment:
Do we need to do this? Or is this for debugging.

Reply:
The second line was for debugging, removed it